packages/frontend/pom.xml

```diff
@@ -0,0 +1,101 @@
+    <description>Frontend for oTTo</description>
+
+    <properties>
+        <!-- plugin version -->
+        <plugins.openapi-generator.version>6.4.0</plugins.openapi-generator.version>
+    </properties>
+
+    <build>
+        <pluginManagement>
+            <plugins>
+                <!-- source-generation from openapi -->
+                <plugin>
+                    <groupId>org.openapitools</groupId>
+                    <artifactId>openapi-generator-maven-plugin</artifactId>
+                    <version>${plugins.openapi-generator.version}</version>
+                    <executions>
+                        <execution>
+                            <goals>
+                                <goal>generate</goal>
+                            </goals>
+                            <phase>validate</phase>
+                            <configuration>
+                                <inputSpec>${project.basedir}/../api-spec/openapi.yml</inputSpec>
+                                <generatorName>typescript-axios</generatorName>
+                                <configOptions>
+                                    <npmName>rest-client</npmName>
+                                    <supportsES6>true</supportsES6>
+                                    <npmVersion>8.19.3</npmVersion>
+                                    <withInterfaces>true</withInterfaces>
+                                </configOptions>
+                                <output>${project.basedir}/generated-sources/client</output>
+                            </configuration>
+                        </execution>
+                    </executions>
+                </plugin>
+                <plugin>
+                    <groupId>com.github.eirslett</groupId>
+                    <artifactId>frontend-maven-plugin</artifactId>
+                    <!-- Use the latest released version:
+                    https://repo1.maven.org/maven2/com/github/eirslett/frontend-maven-plugin/ -->
+                    <version>1.12.1</version>
+                    <executions>
+                        <execution>
+                            <!-- optional: you don't really need execution ids, but it looks nice in your build log. -->
+                            <id>install node and npm</id>
+                            <goals>
+                                <goal>install-node-and-npm</goal>
+                            </goals>
+                            <!-- optional: default phase is "generate-resources" -->
+                            <phase>generate-resources</phase>
+                        </execution>
+                        <execution>
+                        <id>Build OpenAPI code</id>
+                        <goals>
+                            <goal>npm</goal>
+                        </goals>
+
+                        <!-- optional: default phase is "generate-resources" -->
+                        <phase>generate-resources</phase>
+
+                        <configuration>
```

Comment:
do we need something else then install?
yes -> remove comment
no -> remove arguments-tag

Reply:
As the person who is most likely to work with that in the future, I find it helpful to keep the comments